deny.toml

```diff
@@ -46,6 +46,7 @@ allow = [
     "ISC",
     "MIT",
     "MPL-2.0",
+    "OpenSSL",
```

Comment:
This license was recently removed in https://github.com/neondatabase/neon/pull/11693.
Hence, likely fine to add it back now that we need it again.